Library/OcDebugProtocolLib/OcDebugProtocolLib.c

```diff
@@ -25,17 +25,37 @@
 #include <Library/PrintLib.h>
 #include <Library/UefiBootServicesTableLib.h>
 
-#include "OcLogInternal.h"
+STATIC
```

Comment:
Is there any better approach than duplicating?

Reply:
I would say better not to.

Reply:
Not to?